spec/policies/proposal_policy_spec.rb

```diff
@@ -162,9 +162,21 @@
       expect(ProposalPolicy).not_to permit(proposal.requester, proposal)
     end
 
-    it "doesn't allow an approver to cancel it" do
-      proposal = create(:proposal, :with_parallel_approvers, :with_observers)
-      expect(ProposalPolicy).not_to permit(proposal.approvers[0], proposal)
+    it "allows an actionable approver to cancel it" do
+      proposal = create(:proposal, :with_serial_approvers, :with_observers)
+      expect(ProposalPolicy).to permit(proposal.approvers[0], proposal)
+    end
+
+    it "allows a delegate of an actionable approver to cancel it" do
```

Comment:
we have a transient `delegate` attr on the `proposal` factory so you could shorten this:

```ruby
delegate = create(:user)
proposal = create(:proposal, delegate: delegate)
expect(ProposalPolicy).to permit(delegate, proposal)
```